docs/nada.md

```diff
@@ -373,23 +445,25 @@ def nada_main():
 Here is the generated protocols model text file for the 3 Party addition program above:
 
 ```
+Literals:
 Inputs:
 
+
 iaddr(0): A (ref 0) (sizeof: 1)
 iaddr(1): B (ref 0) (sizeof: 1)
 
 
-Literals:
-
+Outputs:
+addr(4): my_output
 
-Protocols:
-rty(ShamirShareInteger) = P2S iaddr(0)                                                              # result = a + b  -> main.py:10
-rty(ShamirShareInteger) = P2S iaddr(1)                                                              # result = a + b  -> main.py:10
-rty(ShamirShareInteger) = ADD addr(2) addr(3)                                                       # result = a + b  -> main.py:10
 
+Execution plan:
+Execution step 0 [Local]:
+    rty(ShamirShareInteger) = P2S iaddr(0)                                                              # result = a + b  -> main.py:10
+    rty(ShamirShareInteger) = P2S iaddr(1)                                                              # result = a + b  -> main.py:10
+    rty(ShamirShareInteger) = ADD addr(2) addr(3)                                                       # result = a + b  -> main.py:10
+Execution step 0 [Online]:
```

Comment:
I'm missing some explanation about the format of this output. What is `rty(ShamirShareInteger) = P2S iaddr(0)`? If I remember, the line says something like: 
"In the memory address 0 we have a Particle-to-Share protocol for an entry with runtime type `ShamirShareInteger`. "

Also we should say that this format is experimental. I'm not sure it will stay like this forever. For instance, we will need to list all the protocol abbreviations. For instance, we need to define what is `ADD` and what is the actuall protocol it corresponds to. 

Finally, could you pick up and example that does not require the use of the particle to share protocol please?

Reply:
I see, I'll update the result.